decidim-assemblies/app/views/decidim/assemblies/admin/assemblies/index.html.erb

```diff
@@ -77,15 +77,11 @@
                <% else %>
                  <span class="action-space icon"></span>
                <% end %>
-               <% if assembly.children.count.positive? || allowed_to?(:create, :assembly) %>
-                 <%= icon_link_to "government-line",
-                                  url_for(query_params_with(parent_id_eq: assembly.id)),
-                                  t("decidim.admin.titles.assemblies"),
-                                  class: "action-icon--dial #{"highlighted" if assembly.children.count.positive?}" %>
-               <% else %>
-                 <span class="action-space icon"></span>
-               <% end %>
-              <% if allowed_to? :copy, :assembly, assembly: assembly %>
+               <%= icon_link_to "government-line",
```

Comment:
Why changing the way the if is being implemented?  Also, I guess here we could do a check and see if the user is allowed to browse those assemblies (like a `allowed_to?(:read, :assembly, assembly: )`) ?